library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -956,6 +962,15 @@ public Builder textColor(int textColor) {
             return this;
         }
 
+        public Builder typeFace(String fontPath) {
+            this.fontPath = fontPath;
```

Comment:
Keep a standard for names. You called the function `typeFace`, but internally the variable was called `fontPath`.

However, I think the best thing would be to receive `Typeface` directly and not `String`.

```java
        public Builder typeface(Typeface typeface) {
            this.typeface = typeface;
        }
```